labkey-client-api/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # The LabKey Remote API Library for Java - Change Log
 
+## version x.y.z
+*Released*: ??
```

Comment:
Since this is the only change we'll be publishing with the next version, please update this to version 1.5.2 with today as the release date before merging.